programs/vote/src/vote_state/mod.rs

```diff
@@ -1037,13 +1003,9 @@ pub fn initialize_account<S: std::hash::BuildHasher>(
     vote_init: &VoteInit,
     signers: &HashSet<Pubkey, S>,
     clock: &Clock,
-    feature_set: &FeatureSet,
```

Comment:
keeping the plumbing in for the inevitable future votestate / vote format upgrade 🙃